bin/update_timestamp

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
+
+require File.expand_path("../../config/environment", __FILE__)
+require "specialist_publisher"
+
+def update
+  $stderr.puts %Q{
+USAGE: #{File.basename(__FILE__)} (all|[format ...])(content_id) (date)
+
+1. available formats:
+
+  #{SpecialistPublisher.document_types.sort.join("\n  ")}
+
+2. content_id:
+	To find content_id, login to specialist-publisher	in production, go to content item page and copy content_id from url.
```

Comment:
I'm not sure if this is the right place for the advice on how to find the `content_id`. It might be, but previously that kind of information's been written up in the Ops Manual's Specialist Publisher section (https://github.gds/pages/gds/opsmanual/2nd-line/applications/specialist-publisher.html)